README.md

```diff
@@ -370,7 +370,7 @@ use {
   -- The setup key implies opt = true
   setup = string or function,  -- Specifies code to run before this plugin is loaded.
   -- The following keys all imply lazy-loading and imply opt = true
-  cmd = string or list,        -- Specifies commands which load this plugin.
+  cmd = string or list,        -- Specifies commands which load this plugin. Can have patterns.
```

Comment:
I think this wording should be the same as the one in `packer.txt` as that is clearer